fizzbuzz_class.rb

```diff
@@ -0,0 +1,19 @@
+class FizzBuzzClass
```

Comment:
Create an initialize method for this, which calls `generate_fizzbuzz` and then assigns the resulting array to the attribute `fizzbuzz_numbers`